apps/scratch/src/types/scratch-student-activities/create/types.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  BaseStudentAppActivity,
+  BaseActivityRequest,
+  StudentAction,
+} from "../common";
+
+export interface StudentCreateActivity extends BaseStudentAppActivity {
+  parentId: string | null;
```

Comment:
I would also add a tsdoc saying that it is null if there is no parent / this is a top-level block (assuming that this is the case ^^)